### ISSUE TITLE
AcidTask high CPU usage and cause server lag 25+ players

### ISSUE THREAD

**Issue by andris155** [BentoBoxWorld/AcidIsland#92]
**Description**
AcidTask high CPU usage and cause server lag.
CPU: Intel Core i9-9900K

**Steps to reproduce the behavior:**
Use AcidIsland addon with 25+ players.

**Expected behavior**
AcidTask no high CPU usage and no cause server lag.

**Screenshots**
<img width="1164" alt="Capture" src="https://user-images.githubusercontent.com/8324198/89716822-9c6bbf80-d9b0-11ea-821f-9a5c535f8429.PNG">

**Server Information:**

[Please complete the following information:]
 - Database being used (Flat file, MySQL, Mongo): JSON
 - OS: Debian 10
 - Java Version: Java 11.0.7
 - BentoBox version: 1.15.0-SNAPSHOT-b1846
 - Addons installed?
```
[19:47:14 INFO]: AcidIsland 1.14.1 (ENABLED)
[19:47:14 INFO]: Biomes 1.13.0-SNAPSHOT-b180 (ENABLED)
[19:47:14 INFO]: Challenges 0.8.4-SNAPSHOT-b428 (ENABLED)
[19:47:14 INFO]: Chat 1.1.0-SNAPSHOT-b30 (ENABLED)
[19:47:14 INFO]: DimensionalTrees 1.6.1-SNAPSHOT-b81 (ENABLED)
[19:47:14 INFO]: Greenhouses 1.2.0-SNAPSHOT-b242 (ENABLED)
[19:47:14 INFO]: Level 2.3.4-SNAPSHOT-b403 (ENABLED)
[19:47:14 INFO]: Likes 2.1.0-SNAPSHOT-b29 (ENABLED)
[19:47:14 INFO]: MagicCobblestoneGenerator 2.1.0-SNAPSHOT-b154 (ENABLED)
[19:47:14 INFO]: Warps 1.10.0 (ENABLED)
```
 - Other plugins?

`Plugins (91): AdminVedelem, AdvancedAchievements, antiRedstoneClock, AnvilRestrict, ArmorStandEditor, BentoBox, BetterGUI, BungeeHelpop, BuntetesLogolo, ChatControl, ChatReaction, ChestShop, ChestShopNotifier*, ChestSort, Citizens, CompatNoCheatPlus, ConditionalCommands, CoreProtect, CraftBook, CratesPlus, CrazyAuctions, CrazyAuctionsLog, CS-CoreLib, CustomCrafting, Duels, DyedBackpacks, EcoPower, EnderContainers, EntityDetection, Essentials, EssentialsChat, EssentialsSpawn, EventekGUI, EventRendszer, ExtraGear, ExtraTools, HeadDatabase*, HolographicDisplays, Jatekido, Kemkedes, Kezdes, LockettePro, Lottery, LuckPerms, mcMMO, MineMarket-Universal, MobFarmManager, Multiverse-Core, Multiverse-NetherPortals, MyCommand, MyPet, NametagEdit*, NoCheatPlus, OpenInv, PickupSpawners, PlaceholderAPI, PlugMan, PocketGames, PomaBot, PomaDrops, PotionArmors, ProtocolLib, RedstoneClockDetector, RemoveUnbreaking, RestrictedCreative, ShopGUIPlus, Shopkeepers, SkinsRestorer, sleep-most*, Slimefun, SlimefunLuckyBlocks, SlimefunPomaFix, StaffChat, StaffTime, SzulinapiKalapok, TimeIsMoney*, TitleManager, TokenManager, TokenShopEditor, TokenStaff*, TreasureChest, Vault, Votifier, VotingPlugin, WDLCompanion*, WolfyUtilities, WorldBorder, WorldEdit, WorldGuard, WorldGuardExtraFlags, WorldGuardPomaFlags
`
**Additional context**
https://timings.aikar.co/?id=715b00e257764d4ab7087f68b6e0476c

**Comment by tastybento** [comment]
I had a look at the timing you shared. I'll take your word for it that you're seeing lag, but across the period that the timings are taken, your TPS reports steady at 20.
<img width="977" alt="Screen Shot 2020-08-08 at 12 41 09 PM" src="https://user-images.githubusercontent.com/4407265/89718577-7b857900-d974-11ea-8da1-da82276710c9.png">

Anyway, please share your config.yml (zip it and drop it here).
Do you have anything set for:
```
      # Destroy items after this many seconds in acid. 0 = do not destroy items
      item: 0
```
I suspect it is set to 1? There are two acid tasks running - one is to burn entities, the other is to remove items. You can increase this value so it doesn't run so often.

Let me know.

**Comment by andris155** [comment]
Yes with 10 players TPS is 20.

**Comment by andris155** [comment]
[config.zip](https://github.com/BentoBoxWorld/AcidIsland/files/5046163/config.zip)

**Comment by tastybento** [comment]
This this and see how it works for you. I made some of the task async.
[AcidIsland-1.14.2-SNAPSHOT-LOCAL.jar.zip](https://github.com/BentoBoxWorld/AcidIsland/files/5046707/AcidIsland-1.14.2-SNAPSHOT-LOCAL.jar.zip)

**Comment by andris155** [comment]
This jar totally fixed this.

**Comment by andris155** [comment]
<img width="1121" alt="Capture" src="https://user-images.githubusercontent.com/8324198/89731711-86f6a400-da49-11ea-8a6d-d2cfc955c53d.PNG">

**Comment by tastybento** [comment]
Implemented in c1515e8689b732a540c060f342b022760d4521ee

**Comment by andris155** [comment]
Very big thanks!
